### PR TITLE
Returning to previous lint version

### PR DESCRIPTION
**Pull request by swiftgist**
This restores master to a clean lint and working tox.ini

Signed-off-by: Eric Jackson <ejackson@suse.com>

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -9,30 +9,23 @@ envlist = py27,lint
...
     salttesting
     configobj
     boto
-    codecov
-    pytest-cov
 
 [testenv:lint]
-basepython = python3
-commands = pylint --rcfile=.pylintrc  srv/
+commands = pylint --rcfile=.pylintrc --jobs=5 srv/
```
Comment (kbaikov) [inline tox.ini]: I believe jobs parameter is redundant here since it is already in pylintc

File: tox.ini [hunk 2]
```diff
@@ -9,30 +9,23 @@ envlist = py27,lint
 [base]
 deps =
     ipaddress
-    salt
+    salt<2017.0.0
 
 [testenv:py27]
-basepython = python2.7
```
Comment (kbaikov) [inline tox.ini]: Why are you removing explicit py versions from here and from lint?

**Comment by kshtsk** [comment]
retest this please

**Comment by jschmid1** [comment]
Running make lint on the current master gives me only one single error.

```
************* Module srv.salt._modules.deepsea
W: 14,20: Unused variable 'state2' (unused-variable)
```

File: tox.ini (hunk 1 above)
Comment (swiftgist) [reply tox.ini]: I simply pulled this from SES5 in order to get environments working.  If we have been moving parameters from command line to config file, I am not aware.

File: tox.ini (hunk 2 above)
Comment (swiftgist) [reply tox.ini]: Same as above.  It's not deletion as much as restoring to a known working state.

**Comment by swiftgist** [comment]
Since #1186 works for me, this is not needed.

**Pull request closed without merging**